hack/configapi/cmd/root.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/edgelesssys/constellation/v2/internal/api/configapi"
+	"github.com/edgelesssys/constellation/v2/internal/staticupload"
+	"github.com/spf13/cobra"
+)
+
+const (
+	awsRegion      = "eu-central-1"
+	awsBucket      = "cdn-constellation-backend"
+	invalidDefault = 0
+	envAwsKeyID    = "AWS_ACCESS_KEY_ID"
+	envAwsKey      = "AWS_ACCESS_KEY"
+)
+
+var (
+	versionFilePath string
+	// Cosign credentials.
+	cosignPwd      string
+	privateKeyPath string
+)
+
+// Execute executes the root command.
+func Execute() error {
+	return newRootCmd().Execute()
+}
+
+// newRootCmd creates the root command.
+func newRootCmd() *cobra.Command {
+	uploadCmd := &cobra.Command{
+		Use:   "AWS_ACCESS_KEY_ID=$ID AWS_ACCESS_KEY=$KEY upload -b 2 -t 0 -s 6 -m 93 --cosign-pwd $PWD --private-key $FILE_PATH",
+		Short: "Upload a set of versions specific to the azure-sev-snp attestation variant to the config api.",
+
+		Long: "Upload a set of versions specific to the azure-sev-snp attestation variant to the config api. Please authenticate with AWS through your preferred method (e.g. environment variables, CLI) to be able to upload to S3.",
+		RunE: runCmd,
+	}
+	uploadCmd.PersistentFlags().StringVarP(&versionFilePath, "version-file", "f", "", "File path to the version json file.")
+
+	uploadCmd.PersistentFlags().StringVar(&cosignPwd, "cosign-pwd", "", "Cosign password used to decrpyt the private key.")
+
+	uploadCmd.PersistentFlags().StringVar(&privateKeyPath, "private-key", "", "File path of private key used to sign the payload.")
+	return uploadCmd
+}
+
+func runCmd(cmd *cobra.Command, _ []string) error {
+	if err := enforceRequiredFlags(cmd, "version-file", "cosign-pwd", "private-key"); err != nil {
+		return err
+	}
+	ctx := cmd.Context()
+	cfg := staticupload.Config{
+		Bucket: awsBucket,
+		Region: awsRegion,
+	}
+	privateKey, err := getBytesFromFilePath(privateKeyPath)
+	if err != nil {
+		return fmt.Errorf("reading private key: %w", err)
+	}
+
+	versionBytes, err := getBytesFromFilePath(versionFilePath)
```

Comment:
```suggestion
	versionBytes, err := os.ReadFile(versionFilePath)
```
And then you can remove the `geBytesFromFilePath` function on line 103